projects/plugins/jetpack/modules/likes/jetpack-likes-settings.php

```diff
@@ -652,8 +653,17 @@ public function is_comments_enabled() {
 	public function admin_settings_callback() {
 		// We're looking for these, and doing a dance to set some stats and save
 		// them together in array option.
-		$new_state         = ! empty( $_POST['wpl_default'] ) ? sanitize_text_field( wp_unslash( $_POST['wpl_default'] ) ) : 'on'; // phpcs:ignore WordPress.Security.NonceVerification.Missing -- no site changes.
-		$reblogs_new_state = ! empty( $_POST['jetpack_reblogs_enabled'] ) ? sanitize_text_field( wp_unslash( $_POST['jetpack_reblogs_enabled'] ) ) : 'on'; // phpcs:ignore WordPress.Security.NonceVerification.Missing -- no site changes.
+		if ( ! empty( $_POST['wpl_default'] ) && isset( $_POST['_wpnonce'] ) && wp_verify_nonce( $_POST['_wpnonce'], 'sharing-options' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput -- WordPress core doesn't unslash or verify nonces either.
+			$new_state = sanitize_text_field( wp_unslash( $_POST['wpl_default'] ) );
+		} else {
+			$new_state = 'on';
+		}
+
+		if ( ! empty( $_POST['jetpack_reblogs_enabled'] ) && isset( $_POST['_wpnonce'] ) && wp_verify_nonce( $_POST['_wpnonce'], 'jetpack-reblog-option' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput -- WordPress core doesn't unslash or verify nonces either.
```

Comment:
Why are you checking the nonce again with a different action? That's never going to work.

And trying it on wpcom (I'm not sure how to get there clicking, maybe it's hidden in Calypso instead, but going directly to `/wp-admin/options-general.php?page=sharing` on my sandbox works) I find that I can no longer change this setting, it's always set to "on".